tests/framework/validators/IpValidatorTest.php

```diff
@@ -0,0 +1,190 @@
+<?php
+namespace yiiunit\framework\validators;
+
+use yii\validators\IpValidator;
+use yiiunit\data\validators\models\FakedValidationModel;
+use yiiunit\TestCase;
+
+/**
+ * @group validators
+ */
+class IpValidatorTest extends TestCase
```

Comment:
The test fails when ipv6 support isn't there. Should be marked as skipped in the case instead.